source/common/config/utility.cc

```diff
@@ -308,5 +308,31 @@ void Utility::translateOpaqueConfig(const ProtobufWkt::Any& typed_config,
   }
 }
 
+JitteredExponentialBackOffStrategyPtr Utility::buildJitteredExponentialBackOffStrategy(
+    absl::optional<const envoy::config::core::v3::BackoffStrategy> backoff,
+    Random::RandomGenerator& random, const uint32_t default_base_interval_ms,
+    absl::optional<const uint32_t> default_max_interval_ms) {
+  // BackoffStrategy config is specified
+  if (backoff != absl::nullopt) {
+    uint32_t base_interval_ms = PROTOBUF_GET_MS_REQUIRED(backoff.value(), base_interval);
+    uint32_t max_interval_ms =
+        PROTOBUF_GET_MS_OR_DEFAULT(backoff.value(), max_interval, base_interval_ms * 10);
+
+    if (max_interval_ms < base_interval_ms) {
+      throw EnvoyException("max_interval must be greater than or equal to the base_interval");
+    }
+    return std::make_unique<JitteredExponentialBackOffStrategy>(base_interval_ms, max_interval_ms,
+                                                                random);
+  }
+  // default maximum interval is specified
```

Comment:
nit: Add an assertion constraint for the default_base_interval_ms (similar to the PGV constraint for `base_interval` in `BackoffStrategy`.

Reply:
done, thanks